workflow/run_simulation.rb

```diff
@@ -125,6 +126,11 @@ def run_workflow(basedir, rundir, hpxml, debug, timeseries_output_freq, timeseri
     options[:timeseries_time_column_types] << t
   end
 
+  options[:timeseries_output_variables] = []
+  opts.on('--add-timeseries-output-variable NAME', 'Add timeseries output variable; can be called multiple times') do |t|
+    options[:timeseries_output_variables] << t
+  end
```

Comment:
I tried this new option but realized it only works if you request a standard timeseries output too because the timeseries frequency needs to be set. For example, I tried `openstudio workflow/run_simulation.rb -x workflow/sample_files/base.xml -o test --add-timeseries-output-variable 'Site Total Sky Cover'` and no timeseries output file was produced.

Any ideas on how to address this?

If I had to do over, I would replace all the `--hourly`, `--daily`, etc with a `--timeseries-frequency` and `--add-timeseries-output-type` or something. That would have been smarter. But I'd prefer not to make a breaking change at this point.

Reply:
Just assume hourly timeseries (with no output types) when output variables are requesting but a frequency is not specified.

Reply:
Good idea. I'm going to switch it to `timestep` instead of `hourly`, which will obviously be the same thing for hourly simulations but seems like a better default to me.